sdk/storage/azure-storage-blob-changefeed/samples/change_feed_samples.py

```diff
@@ -0,0 +1,108 @@
+# coding: utf-8
+
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+"""
+FILE: change_feed_samples.py
+DESCRIPTION:
+    This sample demonstrates
+    1) list events by page
+    2) list all events
+    3) list events in a time range
+    4) list events starting from a continuation token
+USAGE:
+    python blob_samples_container.py
+    Set the environment variables with your own values before running the sample:
+    1) AZURE_STORAGE_ACCOUNT_NAME - the name of the storage account
+    2) AZURE_STORAGE_ACCESS_KEY - the storage account access key
+"""
+
+import os
+from datetime import datetime
+from azure.storage.blob.changefeed import ChangeFeedClient
+
+
+class ChangeFeedSamples(object):
+
+    ACCOUNT_NAME = os.getenv("AZURE_STORAGE_ACCOUNT_NAME")
+    ACCOUNT_KEY = os.getenv("AZURE_STORAGE_ACCESS_KEY")
+
+    def list_events_by_page(self):
+
+        # Instantiate a ChangeFeedClient
+        # [START list_events_by_page]
+        # [START create_change_feed_client]
+        cf_client = ChangeFeedClient("http://{}.blob.core.windows.net".format(self.ACCOUNT_NAME),
+                                     credential=self.ACCOUNT_KEY)
+        # [END create_change_feed_client]
+
+        change_feed = cf_client.list_changes(results_per_page=10).by_page()
+
+        # print first page of events
+        change_feed_page1 = next(change_feed)
+        events_per_page = list(change_feed_page1)
+        for event in events_per_page:
+            print(event)
+
+        # print second page of events
+        change_feed_page2 = next(change_feed)
+        events_per_page = list(change_feed_page2)
+        for event in events_per_page:
+            print(event)
+        # [END list_events_by_page]
+
+    def list_all_events(self):
+        # [START list_all_events]
+        cf_client = ChangeFeedClient("http://{}.blob.core.windows.net".format(self.ACCOUNT_NAME),
+                                     credential=self.ACCOUNT_KEY)
+        change_feed = cf_client.list_changes()
+
+        # print all events
+        events = list(change_feed)
```

Comment:
We should remove this line - we don't want to encourage pre-downloading the generator, as for some people this could be really slow.

Reply:
In fact we can remove the `list()` casting in each of the samples.